docs/getting_started/basics/authoring.md

```diff
@@ -5,15 +5,15 @@ When launching Shift, an new empty board is opened where nodes and connections c
 
```

Comment:
Just a question, do we have at any point of the documentation what is the main focus on using the trigger plugs? I don't find any reference and I think it's important to explain them.

Reply:
Yes, it is explained in the Node's page

Reply:
Ohhh okay, yes, I didn't find that page. Thanks!